### PR TITLE
TOMEE-2575 - translate of examples/cdi-request-scope

### PR DESCRIPTION
**Pull request by gabrielferreirapro**
Hey guys there is my translation of cdi-request-scope.
Thank you ;)
jira task: https://issues.apache.org/jira/browse/TOMEE-2575

### CODE REVIEW

File: examples/cdi-request-scope/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+== Exemplo
+
+Este exemplo retrata um cenário similar ao cdi-application-scope. O visitante de um restaurante 
+pede uma sopa ao garçom. O pedido é passado para o chefe que prepara o prato e o devolve ao garçom 
+que entrega para o visitante.
+
+=== Waiter (garçom)
+
+O `Waiter` session bean recebe uma requisição através do método `orderSoup()` da classe de teste.
+Uma instancia de `Soup` sera criada nesse método e ela sera compartilhada em toda a requisição com o
```
Comment (marcoantoniobferreira) [inline examples/cdi-request-scope/README_pt.adoc]: Hi Gabriel, in the sentence: "Uma instancia de `Soup` sera criada" the word "instancia" have an accent "instância", and the word "sera" is "será".

File: examples/cdi-request-scope/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        soup.setName(name);
+        return chef.prepareSoup().getName();
+    }
+}
+....
+
+=== Soup (sopa)
+
+A classe `Soup` é um POJO injetável, definida como `@RequestScoped`. Isso significa que uma instância 
+sera criada apenas uma vez para cada request e sera compartilhada por todos os beans que a injetam.
```
Comment (marcoantoniobferreira) [inline examples/cdi-request-scope/README_pt.adoc]: the same question, in the word "sera" = "será"

File: examples/cdi-request-scope/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    @After
+    public void closeContainer() throws Exception {
+        container.close();
+    }
+}
+....
+
+== Executando
+
+Na saida voce pode ver que existem duas instancias de `Soup` criadas - uma para cada requisição.
```
Comment (marcoantoniobferreira) [inline examples/cdi-request-scope/README_pt.adoc]: the word "voce" is "você", and the word "instancias" = "instâncias"

**Comment by gabrielferreirapro** [comment]
> Great translation with some little points of attention =)
Fixed.

Thank you @marcoantoniobferreira for your review, I'm really bad with points and accents.
I will improve this skill.

File: examples/cdi-request-scope/README_pt.adoc (hunk 1 above)
Comment (gabrielferreirapro) [reply examples/cdi-request-scope/README_pt.adoc]: done

**Pull request merged**